pages/_app.pg.tsx

```diff
@@ -8,6 +8,7 @@ import { Header } from "components/Header";
 import { useRouter } from "next/router";
 import { ParallaxProvider } from "react-scroll-parallax";
 import { PageTransition } from "components/PageTransition";
+import { PageLoadProgress } from "../components/PageLoadProgress/PageLoadProgress";
```

Comment:
this project uses **absolute paths** everywhere. Please remove this relative path and use absolute path `components/PageLoadProgress`